src/dialects/oracle/query/compiler.js

```diff
@@ -105,9 +105,19 @@ assign(QueryCompiler_Oracle.prototype, {
   update: function() {
     var updates = this._prepUpdate(this.single.update);
     var where   = this.where();
-    return 'update ' + this.tableName +
-      ' set ' + updates.join(', ') +
-      (where ? ' ' + where : '');
+    var returning = this.single.returning;
+
+    if (!returning) {
+      return 'update ' + this.tableName + ' set ' + updates.join(', ') + (where ? ' ' + where : '');
```

Comment:
@AiNoKame Would you mind declaring this as a seperate variable `sql` and using said variable here and in the call to `_addReturningToSqlAndConvert` ?

Not too fond of using the same string concatenation more than once in a function. Also would make this easier to read.